datacenter/dtr/2.2/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -71,34 +94,67 @@ Where:
 * `--existing-replica-id` is the id of the replica to backup,
 * `--ucp-username`, and `--ucp-password` are the credentials of a UCP administrator.
 
+To avoid having to pass the password as a command line parameter, you may
+instead use the following approach in bash:
+
+```none
+$ read -sp 'ucp password: ' PASS; UCP_PASSWORD=$PASS docker run -i --rm -e UCP_PASSWORD docker/dtr backup \
+  --ucp-url <ucp-url> \
+  --ucp-insecure-tls \
+  --existing-replica-id <replica-id> \
+  --ucp-username <ucp-admin> > /tmp/backup.tar
+```
+
+This puts the password into a shell variable which is then passed into the
+docker client command with the -e flag which in turn relays the password to the
+DTR bootstrapper.
+
+## Testing backups
+
 To validate that the backup was correctly performed, you can print the contents
 of the tar file created:
 
 ```none
 $ tar -tf /tmp/backup.tar
 ```
 
+To really test that the backup works, you must make a copy of your UCP cluster
+by backing it up and restoring it onto separate machines. Then you can restore
+DTR there from your backup and verify that it has all the data you expect to
+see.
+
 ## Restore DTR data
 
-You can restore a DTR node from a backup using the `restore`
-command.
-This command performs a fresh installation of DTR, and reconfigures it with
-the configuration created during a backup.
+You can restore a DTR node from a backup using the `restore` command.
 
-The command starts by installing DTR, restores the configurations stored on
-etcd, and then restores the repository metadata stored on RethinkDB. You
-can use the `--config-only` option, to only restore the configurations stored
-on etcd.
+Before restoring DTR, make sure that you are restoring it on the same UCP
+cluster or you've also restored UCP using its restore command. DTR does not
+manage users, orgs or teams so if you try to
+restore DTR on a cluster other than the one it was backed up on, DTR
+repositories will be associated with users that don't exist and it will appear
+as if the restore operation didn't work.
 
-This command does not restore Docker images. You should implement a separate
-restore procedure for the Docker images stored in your registry, taking in
-consideration whether your DTR installation is configured to store images on
-the filesystem or using a cloud provider.
+Note that to restore DTR, you must first remove any left over containers from
+the previous installation.
 
-You can check the
-[reference documentation](../../reference/cli/backup.md), for the
-backup command to learn about all the available flags.
+The restore command performs a fresh installation of DTR, and reconfigures it with
+the configuration created during a backup. The command starts by installing DTR,
+restores the configurations from the
+backup and then restores the repository metadata. All install time flags are the same
+for restore and apply the same way they would to a new installation.
```

Comment:
I don't really understand what you mean by "All install time flags are the same for restore and apply the same way they would to a new installation." Are you trying to explain that all configurations will be restored?